scripts/zones/Tahrongi_Canyon/npcs/Telepoint.lua

```diff
@@ -11,16 +11,21 @@ require("scripts/globals/npc_util")
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    -- trade any normal crystal for a faded crystal
+    -- Trade any normal crystal for a faded crystal.
     local item = trade:getItemId()
-    if trade:getItemCount() == 1 and item >= xi.items.FIRE_CRYSTAL and item <= xi.items.DARK_CRYSTAL and npcUtil.giveItem(player, xi.items.FADED_CRYSTAL) then
+    if
+        trade:getItemCount() == 1 and
+        item >= xi.items.FIRE_CRYSTAL and
+        item <= xi.items.DARK_CRYSTAL and
+        npcUtil.giveItem(player, xi.items.FADED_CRYSTAL)
+    then
         player:tradeComplete()
     end
 end
 
 entity.onTrigger = function(player, npc)
     if not player:hasKeyItem(xi.ki.MEA_GATE_CRYSTAL) then
-        npcUtil.giveKeyItem(player, xi.ki.MEA_GATE_CRYSTAL)
+        player:startEvent(32)
     else
```

Comment:
Was curious on this one, because I didn't recall a CS.  Confirmed just a minute ago that it it actually an event:
![image](https://user-images.githubusercontent.com/81713309/153777956-854535d1-bc95-4af7-a2c8-238abe25cf6d.png)

Reply:
give away that its an event: in game if you face slightly away, the game forces your character to pivot before displaying the message

sneaky lil behaviors on retail

Reply:
`La_Theine_Plateau/npcs/Telepoint.lua` needs same treatment too.

Reply:
@Xaver-DaRed Which points are you still missing? Can grab the data.

Reply:
east altepa, yhoatot jungle and 2 from the past, jugner forst S and tarongi cannon S